NitroxServer/Server.cs

```diff
@@ -69,7 +70,25 @@ public void Save()
 
             IsSaving = true;
             NitroxConfig.Serialize(serverConfig); // This is overwriting the config file => server has to be closed before making changes to it
-            worldPersistence.Save(world, serverConfig.SaveName);
+            if (worldPersistence.Save(world, serverConfig.SaveName))
+            {
+                try
+                {
+                    string postSaveCommandPathTrimmed = serverConfig.PostSaveCommandPath.Trim('"');
+
+                    // Call external tool for backups, etc
+                    if (!string.IsNullOrEmpty(postSaveCommandPathTrimmed) ||
+                        File.Exists(postSaveCommandPathTrimmed))
+                    {
+                        Process.Start(postSaveCommandPathTrimmed);
+                        Log.Info($"Post-save command completed successfully: {postSaveCommandPathTrimmed}");
+                    }
+                }
+                catch (Exception Ex)
```

Comment:
```suggestion
                catch (Exception ex)
```
Lowercase var names.